src/entries/popup/pages/home/Activity/ActivityDetails.tsx

```diff
@@ -288,13 +293,40 @@ function NetworkData({ transaction: tx }: { transaction: RainbowTransaction }) {
   );
 }
 
+function SpeedUpErrorExplainer() {
+  const [searchParams, setSearchParams] = useSearchParams();
+  const explainer = searchParams.get('explainer');
+
+  return (
+    <ExplainerSheet
+      show={explainer === 'speed_up_error'}
+      onClickOutside={() => setSearchParams({})}
+      header={{
+        icon: <Symbol symbol="xmark.circle.fill" color="red" size={32} />,
+      }}
+      title="Failed to speed up transaction"
+      description={[
+        'A speed up can fail when the state of the network changed since you first submitted the transaction',
```

Comment:
@DanielSinclair let's define a copy for this and I add to the translations

Reply:
@greg-schrammel Some copy:

"Speed up failed"

"Attempts to speed up a transaction can fail during network congestion, gas limit increases, or when the original transaction is confirmed first"